tools/deployment_manager/configs/linux.json

```diff
@@ -22,28 +22,33 @@
     "ie_core": {
       "group": ["ie"],
       "files": [
-        "runtime/lib/intel64/libopenvino.so.2022.3.0",
-        "runtime/lib/intel64/libopenvino.so.2230",
+        "runtime/lib/intel64/libopenvino.so.2023.0.0",
+        "runtime/lib/intel64/libopenvino.so.2300",
```

Comment:
such paths don't work for arm64.
let's enable arm64 support as well

Reply:
I'll proceed with new PR for introducing arm64